examples/react-native-expo/metro.config.js

```diff
@@ -0,0 +1,14 @@
+const path = require('path');
+const {getDefaultConfig} = require('expo/metro-config');
+
+const root = path.resolve(__dirname, '../..');
+const config = getDefaultConfig(__dirname);
```

Comment:
🟡 since we change the rechunk.json when we start the dev-server etc. we would want to invalidate the cache of metro so babel recalculates transforms:

https://metrobundler.dev/docs/configuration#resetcache

Reply:
oddly enough it doesn't appear that resetCache: true is working for expo